assets/dev/js/admin/admin.js

```diff
@@ -434,10 +435,11 @@ import { showJsonUploadWarningMessageIfNeeded } from 'elementor-utils/json-uploa
 					return;
 				}
 
-				if ( $importFormFileInput[ 0 ].files.length && ! elementorCommon.config.filesUpload.unfilteredFiles ) {
+				if ( $importFormFileInput[ 0 ].files.length && ! elementorCommon.config.filesUpload.unfilteredFiles && ! notEnabledMessageShown ) {
 					event.preventDefault();
 
 					const enableUnfilteredFilesModal = FilesUploadHandler.getUnfilteredFilesNotEnabledImportTemplateDialog( () => {
+						notEnabledMessageShown = true;
 						$importForm.trigger( 'submit' );
```

Comment:
I'm not sure why we need to re trigger the form submit if it is already submitted.

Reply:
We sort-of re-submit after the user approved all warnings.